cypress/integration/happy-path-with-accessibility.js

```diff
@@ -110,24 +109,55 @@ describe('Get help to retrain smoke test', function() {
   });
 
   it('should allow me to choose training options', function() {
+    cy.get('[id="training_english_skills"]').check()
+    cy.get('[id="training_math_skills"]').check()
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
-  it('should allow me to choose computer skills training options', function() {
+  it('should allow me to choose job hunting advice', function() {
+    cy.get('[id="it_training_computer_skills"]').check()   
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
-  it('should allow me to choose job hunting advice and navigate to action plan', function() {
+  it('should allow me to find jobs near me and navigate to action plan', function() {
+    cy.get('[id="job_hunting_cv"]').check();
+    cy.get('[id="job_hunting_cover_letter"').check();
+    cy.get('[id="job_hunting_interviews"]').check(); 
     cy.contains('Continue').click();
+    cy.contains('Show jobs near me').click();
+    checkAccessibility();
+  })
+
+  it('should allow me to find an maths course', function() {
+    cy.go('back')
+    cy.contains('Find a maths course').click()
     checkAccessibility();
   });
 
-  it('should allow me to find jobs near me', function() {
-    cy.contains('Show jobs near me').click();
+  it('should allow me to find an English course', function() {
```

Comment:
do we need to test both english and math, I think one is enough since they are the same page

Reply:
I think it's just the journey from navigating from action plan to another training page which is important here an making sure all is expected as per vanilla scenario.